src/Meta/MySql/Schema.php

```diff
@@ -78,8 +78,9 @@ protected function load()
      */
     protected function fetchTables($schema)
     {
-        $rows = $this->connection->select('SHOW TABLES FROM '.$schema);
-        $tables = Arr::flatten($rows);
+        $rows = $this->connection->select('SHOW FULL TABLES FROM '.$schema.' WHERE Table_type=\'BASE TABLE\'');
```

Comment:
Thanks for contributing :) I've just added a method `wrap` to add backticks to the schema name which seem to have been representing a problem ([#5](https://github.com/reliese/laravel/issues/5)). Please, would you mind rebasing the new changes?